src/doc/conf.py

```diff
@@ -72,9 +72,13 @@ def setup(app):
 
 # Best place to put a substitution for 'VisIt'
 # Authors should use either VisIt_ or VisIt_'s
+# fs*nix = fully supported, ps*nix = partially supported
 rst_epilog = """
 .. _VisIt: https://visit.llnl.gov
 .. _Silo: https://silo.llnl.gov
+.. |*nix| replace:: Centos, Debian, Fedora, Redhat, SUSE, TOSS, Ubuntu
+.. |fs*nix| replace:: Redhat, TOSS, Ubuntu
+.. |ps*nix| replace:: Centos, Debian, Fedora, SUSE
```

Comment:
I haven't done anything with SUSE, so you may want to remove it from this list (and maybe the one above).